torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1929,6 +1931,16 @@ def hint():
                     source == symbol_to_source[expr][0]
                 ):
                     continue
+
+                # This logic excludes static values found on tensors from guarding, because
+                # dynamo's check_tensor_fn does that (see guards.cpp).
+                # However, for non tensor sources, we still need to guard here.
+                if ignore_static and isinstance(source, TensorPropertySource):
+                    maybe_static = self._maybe_evaluate_static(expr)
+                    if isinstance(maybe_static, (int, sympy.Integer)):
+                        self.log.info("Skipping guard %s", f"{source_ref(source)} == {maybe_static}")
```

Comment:
I think this is too chatty for info. Suggest debug.

Reply:
sg